projects/appearances/protos/Pavement.proto

```diff
@@ -0,0 +1,49 @@
+#VRML_SIM R2020a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# A pavement material. The type can be choose between "old_small", "old_big", "standard", "slate", "regular" and "grid". The color can be overridden using the `colorOverride` field.
+
+PROTO Pavement [
+  field SFString{"old_small", "old_big", "standard", "slate", "regular", "grid"} type             "standard" # Defines the pavement type.
```

Comment:
you should avoid `_` just use spaces (they are replaced by the `gsub` a few line later).